db/migrate/20220324220850_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- [ ]  It will be nice to replace `:users` with `:author` as the foreign_key. Similar reason is given above.